lib/server.js

```diff
@@ -71,7 +83,7 @@ Server.prototype.listen = function listen(port, address, callback) {
                 };
 
                 var src = {
-                        family: 'udp6',
+                        family: fam,
```

Comment:
(I *think* this will match the socket's type, but you could also make this `rinfo.family === "IPv6" ? 'udp6' : "udp4"`.)